src/components/Toast/types.ts

```diff
@@ -16,6 +16,7 @@ export interface ToastProps extends TPAComponentProps {
   shouldShowCloseButton?: boolean;
   /** Callback function, will be called when click on close button */
   onClose?(event: MouseEvent): void;
+  placement?: 'inline' | 'bottomFullWidth';
```

Comment:
Please change this to an enum

Reply:
No problem. I just should to mention, that 
```ts
enum E {
  p = 'stringLiteral'
}

let prop: E = 'stringLiteral'  // cannot assign ts error
```
this means, that ts users can't use component without importing enum. 
Is it expected?

Reply:
> The intent of string enums is that the values of the enum are opaque (i.e. they can be changed behind the scenes without breaking consumers).

Using enums for theme and same simple cases, where value would not change in future, does not make much sense.

Reply:
Yeah, so first of all, it's pretty much a convention we have in the library, and I prefer keeping it aligned.
Second, I respectfully disagree with this view of enums.
We usually provide consumers with enums to use for props.
This allows javascript users to receive the enum (as an object).
I see typescript as a helpful guide while developing, but not changing the models and architecture of our code.
If it was javascript, we would have used an object to simulate an enum.

Reply:
done